spec/interactors/send_password_reset_email_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe SendPasswordResetEmail do
+  describe ".call" do
+    let(:user) { create(:confirmed_user) }
+
+    subject(:send_email) do
+      described_class.call(user: user)
+    end
+
+    before do
+      CreatePasswordResetToken.call(user: user)
+      ActionMailer::Base.deliveries.clear
+    end
+
+    context "when successful" do
+      it "sends a password reset email" do
+        send_email
+        expect(ActionMailer::Base.deliveries.count).to eq(1)
```

Comment:
Is this the appropriate thing to test? It seems that I'm testing `UserMailer.password_reset` in addition to the `.call` method. Perhaps I should just check to see that `UserMailer.password_reset` receives `deliver_now`?